insuree/services.py

```diff
@@ -329,7 +349,8 @@ def create_or_update(self, data, create_only=False):
         status = data.get('status', InsureeStatus.ACTIVE)
         if status not in [choice[0] for choice in InsureeStatus.choices]:
             raise ValidationError(_("mutation.insuree.wrong_status"))
-        if InsureeConfig.is_insuree_photo_required and photo_data is None:
+        # If photo is set it cannot be deleted 
+        if InsureeConfig.is_insuree_photo_required and photo_data is None and 'uuid' not in data:
```

Comment:
a bit too weak to consider update, create might have uuid in data and update could be done on ID, I think we should rely on create_only=True then photo must be checked (to be adapted in the gql create)